src/objective-c/GRPCClient/private/GRPCReachabilityFlagNames.xmacro.h

```diff
@@ -65,3 +65,4 @@ GRPC_XMACRO_ITEM(interventionRequired, InterventionRequired)
 GRPC_XMACRO_ITEM(connectionOnDemand, ConnectionOnDemand)
 GRPC_XMACRO_ITEM(isLocalAddress, IsLocalAddress)
 GRPC_XMACRO_ITEM(isDirect, IsDirect)
+GRPC_XMACRO_ITEM(isWWAN, IsWWAN)
```

Comment:
I was surprised that a case was missing, but it's hidden above within the `#if TARGET_OS_IPHONE`. It looks like giving it a different name wasn't a good idea :)